hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/ServerRemoteProcedure.java

```diff
@@ -80,6 +81,8 @@ public abstract class ServerRemoteProcedure extends Procedure<MasterProcedureEnv
   protected ServerName targetServer;
   protected boolean dispatched;
   protected boolean succ;
```

Comment:
@Apache9 do you think I should replact this succ flag with state ?

Reply:
Do it in your own convenience. We must have a state here, you can either add a state but still keep this flag, or you can implement all the logic with the state and drop this succ flag.